modules/gapi/src/backends/common/serialization.hpp

```diff
@@ -111,6 +113,16 @@ GAPI_EXPORTS I::IStream& operator>> (I::IStream& is,       cv::gimpl::Protocol &
 GAPI_EXPORTS I::OStream& operator<< (I::OStream& os, const cv::GArg &arg);
 GAPI_EXPORTS I::IStream& operator>> (I::IStream& is,       cv::GArg &arg);
 
+GAPI_EXPORTS I::OStream& operator<< (I::OStream& os, const cv::GMetaArg &arg);
+GAPI_EXPORTS I::IStream& operator>> (I::IStream& is,       cv::GMetaArg &arg);
+
+GAPI_EXPORTS I::OStream& operator<< (I::OStream& os, const cv::GRunArg &arg);
+GAPI_EXPORTS I::IStream& operator>> (I::IStream& is, cv::GRunArg &arg);
+
+GAPI_EXPORTS I::IStream& operator >> (I::IStream& is, cv::GRunArgP &arg);
```

Comment:
No need to export these functions, as those are should be implemented via a generic (template) thing.

Reply:
Still need export GRunArg operators

Reply:
> Still need export GRunArg operators

Hmm no. isn't the header-based generic implementation enough?

Reply:
Done